pkg/eventprocessor/impl/eventprocessor_replayhistory_test.go

```diff
@@ -37,12 +37,12 @@ func TestReplayProductionHistory(t *testing.T) {
 
 	expectedStateHashes := map[tableland.ChainID]string{
```

Comment:
Usually, when we change the parser we break history. In this case, we are now accepting statements that were not being accepted. This means that when we execute historical events we get a different state. That's why we need to update the test state hashes. Only testnets are being updated here, so not a big deal.

Reply:
yes!